CIH_Engine/Query/ValueFromSource.cs

```diff
@@ -75,6 +75,11 @@ private static object GetValue(this IBHoMObject obj, string sourceName)
                     }
 
                 }
+                else
+                {
+                    // Try extracting the property using an Extension method.
+                    BH.Engine.Reflection.Compute.RunExtensionMethod(obj, sourceName);
```

Comment:
```suggestion
                    return BH.Engine.Reflection.Compute.RunExtensionMethod(obj, sourceName);
```

We need a `return` there 😄

Reply:
Sorry, forgot to commit it!